src/app/views/users/_follow.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if current_user && user != current_user %> 
-    <% follows = follows?(current_user, user) %>     
+    <% follows = follows?(user) %>     
```

Comment:
You don't need to assign to a variable anymore you can use `follows?(user)` directly in `link_to_if`